### DISCUSSION TITLE
Issue with Binary caching

### DISCUSSION THREAD

**Discussion by diehard2** [microsoft/vcpkg#30320] [General]
I have binary caching working well locally. I upload to an artifactory server, and I run from a docker container, so everything is always the same. Locally it is always finding the binaries. My issue is on the build system, which is Github actions. Even though I am running the same container, it is only saying 113 of the 140 some libraries are a match and is rebuilding them. I am using 

set(VCPKG_DEFAULT_HOST_TRIPLET x64-linux-dynamic)
set(VCPKG_TARGET_TRIPLET x64-linux-dynamic)

and I am in manifest mode through cmake. I'm not even sure how to go about debugging this on the build system. Or if it even matters since vcpkg has decided the docker image is different when run from there. Is this dependent on processor?

**Comment by diehard2** [comment]
and the 5th time I ran this, it pulled all of them. I guess its transient, but if this does happen again, how do I use --debug when in manifest mode running from cmake?

**Reply by autoantwort** [reply]
You can set [`VCPKG_INSTALL_OPTIONS`](https://learn.microsoft.com/en-us/vcpkg/users/buildsystems/cmake-integration#vcpkg_install_options) to `--debug`
